stages/07.export-stage/03.export-image/run.sh

```diff
@@ -118,6 +118,10 @@ if [ "${CONFIG_RPI_BOOT_FILES}" = y ]; then
 	sed -i "s/ROOTDEV/PARTUUID=${ROOT_PARTUUID}/" "${BUILD_DIR}/boot/cmdline.txt"
 fi
 
+# Configure setup for a specific project and board
+# Pass parameter $BOOTLOADER_DEV in case platfom is Intel
```

Comment:
Actually, you are always passing the parameter, not just for Intel case. It is used just for Intel boot files.
Please update the comment.